package.json

```diff
@@ -56,7 +56,7 @@
     "react-click-outside": "^2.3.0"
   },
   "devDependencies": {
-    "@salesforce-ux/design-system": "2.2.1",
```

Comment:
Should the "^"-character really be there? Maybe use "~"-character for all version 2.3.x.